extensions/resource-deployment/src/services/tools/azCLITool.ts

```diff
@@ -13,7 +13,7 @@ export class AzCliTool implements ITool {
 	}
 
 	get description(): string {
-		return localize('resourceDeployment.AzCLIDescription', 'Tool used for managing Azure services');
+		return localize('resourceDeployment.AzCLIDescription', 'The command-line tool for managing Azure resources');
```

Comment:
nit: Sync with the PMs on this but I think saying "A command-line tool..." works better, since using "the" implies that it's the only one and that may not always be the case.